scripts/sct_compute_mtr.py

```diff
@@ -141,6 +155,12 @@ def get_parser():
                       mandatory=False,
                       example=['0', '1', '2'],
                       default_value='1')
+    parser.add_option(name="-omtr",
+                      type_value="str",
+                      description="Creates output file with the specified path.",
+                      mandatory=False,
+                      example='My_File_Folder/My_New_File',
+                      default_value=os.path.join(os.getcwd(),'mtr'))
```

Comment:
add extension `.nii.gz`

Reply:
agreed, fixed in commit [3b64fb9](https://github.com/YangHee-Min/spinalcordtoolbox/commit/3b64fb92ce0377e63dca16e522411af0ec71ab38)